docs/examples.md

```
@@ -394,5 +394,77 @@ True
+>>> ical_content.count("SUMMARY;LANGUAGE=th:วันสงกรานต์\r\n")
+1
+```
+
+!!! tip "Tip"
+    Although the iCalendar specification supports a wide range of language formats as outlined
+    in [RFC 5646](https://datatracker.ietf.org/doc/html/rfc5646), the Holidays object requires
+    that the `language` attribute adhere to the [ISO 639-1 or
+    ISO 639-2](https://www.loc.gov/standards/iso639-2/php/code_list.php) language codes,
+    such as `en` or `pap-AW`.
+
+    Additionally, if no `language` is specified for a holiday, but a `default_language` is set
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)